blt/deploy/.gitignore

```diff
@@ -1,6 +1,5 @@
 db.sql
 keys
-deploy
```

Comment:
I guess you don't need to ignore the deploy folder in the deploy folder.

Reply:
backstory: theres a `deploy` directory in blt and the deploy commands would fail because those files didnt exist.